src/client/app/containers/LineChartContainer.js

```diff
@@ -25,11 +25,11 @@ function mapStateToProps(state) {
 	}
 
 	for (const meterID of state.graph.selectedMeters) {
-		const readingsData = state.readings.byMeterID[meterID][timeInterval];
+		const readingsData = state.readings.line.byMeterID[meterID][timeInterval];
```

Comment:
I think this is fine. It reads almost like english, and it should be obvious to someone who understands how the redux state is set up.